opendebates/views.py

```diff
@@ -182,6 +182,13 @@ def vote(request, id):
             }
     voter, created = Voter.objects.get_or_create(email=form.cleaned_data['email'])
 
+    if voter.user and voter.user != request.user:
+        # anonymous user can't use the email of a registered user
+        msg = 'That email is registered. Please login and try again.'
+        return HttpResponse(
```

Comment:
oops good catch, will fix. FWIW, i think the view is only called via Ajax currently, but makes sense to fix it properly.